packages/eslint-plugin-ecmascript-compat/lib/targetRuntimes.js

```diff
@@ -3,9 +3,16 @@ const _ = require('lodash');
 const compatData = require('@mdn/browser-compat-data');
 const compareVersions = require('./compareVersions');
 
-module.exports = function targetRuntimes() {
+module.exports = function targetRuntimes(overrideBrowserslist, browserslistOptions) {
   // ['chrome 50', ...]
-  const allNamedVersions = browserslist();
+  const allNamedVersions = browserslist(overrideBrowserslist, browserslistOptions);
+
+  if (allNamedVersions.length === 0) {
+    // eslint-disable-next-line no-console
+    console.warn(
```

Comment:
> I've also added a warning if the browsers list comes back empty - we had missed that when we introduced environments, the tool no longer did anything...

What situation causes it to be empty exactly? If no config (file or package.json) is found at all, it returns defaults.

Reply:
Maybe it’s a corner case. We setup 3 different browser list environments and the default “production” or no environment defined did not exist

shall I remove all comments if that’s the conclusion ?

Reply:
I think remove this warning, as

a) we're having to remove console logs anyway due to console repetition and #76 
b) this would be better handled in browserslist itself, as it's not specific to this plugin